### ISSUE TITLE
[Replay Bug]: a specific website  couldn't replay properly

### ISSUE THREAD

**Issue by cpqcat** [webrecorder/replayweb.page#342]
### ReplayWeb.page Version

v2.0.2

### What did you expect to happen? What happened instead?

I recorded a specific website but it couldn't replay properly.
Normally, after the web progress bar is loaded, the website content will be displayed, but the web I recorded has been stuck in the web  progress bar and the web content will not be displayed.
I have replayed it with the archiveweb.page Extension v0.12.1 and replay v2.0.2. but it still couldn't replay properly.
because the web contains person information, the operation screen capture and wacz has been sent to info@webrecorder.net by 260515242@qq.com.

**Comment by ikreymer** [comment]
I don't believe we received a message from this address, perhaps it didn't go through?

**Comment by cpqcat** [comment]
it seems sent successfully. I sent it to info@webrecorder.net  again  just now by 260515242@qq.com and chenxiaobei8080@gmail.com. the title is "Fw:ArchiveWeb.page bug report"
Could you please check it again?
